src/main/scala/com/lightbend/lagom/docs/BlogMetaDataParser.scala

```diff
@@ -22,7 +22,7 @@ object BlogMetaDataParser {
     def nextOption = if (i.hasNext) Option(i.next()) else None
   }
 
-  private implicit val codec = Codec(Charset.forName("US-ASCII"))
+  private implicit val codec = Codec(Charset.forName("UTF-8"))
```

Comment:
This also broke on the é in Patrick's name.

Originally, it was introduced to ensure that the build works consistently on Windows (Ruth found some problems previously due to unintended curly quotes). Before that, the encoding was not specified at all, so it would have chosen the platform default. Specified consistently as UTF-8, I think it will still avoid Windows problems.

Reply:
👍